web-api/src/persistence/dynamo/cases/getCaseByDocketNumber.ts

```diff
@@ -55,7 +61,14 @@ export const getCaseByDocketNumber = async ({
   }
 
   return purgeDynamoKeys({
-    ...aggregateCaseItems(caseItems),
+    ...aggregateCaseItems([
+      ...caseItems,
+      ...correspondenceItems.map(correspondenceItem => ({
+        ...correspondenceItem,
+        pk: `case|${docketNumber}`,
+        sk: `correspondence|${correspondenceItem.correspondenceId}`,
```

Comment:
We don't need these Dynamo keys anymore. However, it will be easier to get rid of these in the future since `aggregateCaseItems` relies on whatever we are passing in to have the Dynamo keys. Once we have moved everything that `aggregateCaseItems` takes in from Dynamo to Postgres, we can refactor the function and the tests.